app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-05-01"
+
+tabs:
+  opened:
+    type: labeled_counter
+    description: >
+      Number of opened tabs
+    send_in_pings:
+      - session_end
+    labels:
+      - context_menu
+      - tabs_dialog
+      - bookmarks
+      - history
+      - fxa_login
+      - received
+      - pre_existing
+      - browser
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  activated:
+    type: counter
+    description: >
+      Number of activated tabs
+    send_in_pings:
+      - session_end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+firefox_account:
+  sign_in:
+    type: event
+    description: >
+      A user pressed the sign in button on the sync account page
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_success:
+    type: event
+    description: >
+      A user was successfully signed in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_error:
+    type: event
+    description: >
+      An error happened during the login flow and the user couldn't sign in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_out:
+    type: event
+    description: >
+      A user pressed the sign out button on the sync account page and was successfully signed out of FxA
```

Comment:
As for my other comments: are you interested in the counts or are you planning on doing some flow analysis?

Reply:
With the event we also can get the count, right? That's my understanding.

Reply:
Yes, but it's an overkill. Events are meant for analyses in which you care about the order of things (e.g. check if user enabled some mode and *then* tried something else).

Counting is one thing that it's easy to get wrong with events :) Unless you care about sequences of actions, and already have an analysis in line (or Data Science support!), I'd recommend using counts.

Reply:
I think we probably want both, funnel and counts.

Reply:
All right :) One last attempt on my end :P Do you already have an analyst or somebody who can analyse events with great care? If you don't , you might consider adding the counter first, see if you can get the answers you need and, if not, go for events.

This is just for your team's consideration, not really blocking this PR on my end.

Reply:
@daoshengmu I guess that's you at the moment :)

Reply:
Thanks for all your help @Dexterp37 really appreciated!

Reply:
My answer is already posted as above.

Reply:
Changed